content/en/docs/instrumentation/php/manual.md

```diff
@@ -395,7 +395,7 @@ called on the meter provider.
 ### Asynchronous meters
 
 Async meters are `observable`, eg `ObservableGauge`. When registering an
-observable/async meter, you provide one ore more callback functions. The
+observable/async meter, you provide one or more callback functions. The
```

Comment:
I guess that is what  cased somehow

```
[warn] content/en/docs/instrumentation/php/manual.md
[warn] Code style issues found in the above file. Forgot to run Prettier?
[help] Run: npm run format
Error: Process completed with exit code 1.
```
????

Reply:
yeah, it's probably one char to width, run `npm run format` and do a new commit to fix this